packages/bazel/src/ngc-wrapped/index.ts

```diff
@@ -281,6 +280,8 @@ export function compile({
   // Prevent tsickle adding any types at all if we don't want closure compiler annotations.
   if (compilerOpts.annotateForClosureCompiler) {
     bazelHost.transformTypesToClosure = true;
+  }
+  if (compilerOpts.annotationsAs === 'static fields') {
```

Comment:
```suggestion
  if (compilerOpts.annotateForClosureCompiler || compilerOpts.annotationsAs === 'static fields') {
```
To ensure we preserve the previous behavior.

Reply:
I see. I can do that.

Reply:
done